vpr/src/place/simpleRL_move_generator.cpp

```diff
@@ -58,6 +28,60 @@ void SimpleRLMoveGenerator::process_outcome(double reward, e_reward_function rew
  *  K-Armed bandit agent implementation   *
  *                                        *
  *                                        */
+KArmedBanditAgent::KArmedBanditAgent(size_t num_moves, e_agent_space agent_space)
+    : num_available_moves_(num_moves)
+    , propose_blk_type_(agent_space == e_agent_space::MOVE_BLOCK_TYPE) {
+    std::vector<int> available_logical_block_types = get_available_logical_blk_types_();
+    num_available_types_ = available_logical_block_types.size();
+
+    num_available_actions_ = propose_blk_type_ ? (num_available_moves_ * num_available_types_) : num_available_moves_;
+
+    action_logical_blk_type_.clear();
+
+    for (auto logical_blk_type_idx : available_logical_block_types) {
+        action_logical_blk_type_.push_back(logical_blk_type_idx);
+    }
+}
+
+e_move_type KArmedBanditAgent::action_to_move_type_(const size_t action_idx) {
```

Comment:
Comment your action id to move_type, blk_type pair mapping somewhere above here.